includes/class-wc-payments-checkout.php

```diff
@@ -415,16 +415,25 @@ function () use ( $prepared_customer_data ) {
 				);
 			}
 
-			// Output the form HTML.
-			if ( ! empty( $this->gateway->get_description() ) ) : ?>
-				<p><?php echo wp_kses_post( $this->gateway->get_description() ); ?></p>
+			?>
+			<div class="wcpay-upe-form" 
+				data-payment-method-type="<?php echo esc_attr( $this->gateway->get_stripe_id() ); ?>"
+				data-gateway-id="<?php echo esc_attr( $this->gateway->id ); ?>"
+				<?php checked( $this->gateway->chosen, true ); ?>
```

Comment:
Good point. I actually forgot to remove this redundant line because it was me testing how Core is setting the checked gateway when opening the page. This is covered by WooCommerce and its input element (which we're currently actively discussing in another thread) so no need to include that here. Removed in fd1b17cf1f9a01627053e36942ddef6154e4892d